site/src/load.rs

```diff
@@ -213,6 +215,51 @@ impl SiteCtxt {
         )
     }
 
+    /// Returns the not yet tested published artifacts, sorted from newest to oldest.
+    pub async fn missing_artifacts(&self) -> anyhow::Result<Vec<String>> {
```

Comment:
I believe this name is incorrect. "Artifacts" refers to any version of rustc whether identified by tag or commit. But this function just returns the untested released artifacts. So maybe `missing_released_artifacts` is more accurate?

Reply:
Good point.